JSDemos/Demos/Resizable/Overview/description.md

```diff
@@ -1 +1,21 @@
-The Resizable is a UI component that enables its content to be resizable in the UI.
\ No newline at end of file
+The Resizable enables you to resize its content. In this demo you can drag [handles](/Documentation/ApiReference/UI_Components/dxResizable/Configuration/#handles) to resize the DataGrid.
+
+Configure the following properties to specify the boundaries to which you can drag the handles:
```

Comment:
(Concise Text) The word "which" is usually a flag that tells you there might be a way to rewrite the text shorter (or in a more straightforward manner). Here's my suggestion:

```suggestion
Configure the following properties to specify resize operation constraints:
```